lib/translator/translation.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_TRANSLATION_H
+#define LIB_TRANSLATOR_TRANSLATION_H
+
+#include "absl/types/span.h"
+
+#include "common.h"
+
+namespace translator {
+
+// Reports if the user is using the API correctly.
+enum class ApiStatus { kSuccess, kFailure };
+
+struct BeginResponse {
+  ApiStatus status;
+  uint32_t alloc_len;
```

Comment:
I'm guessing this is intended to define the length of `buffer` passed to `Complete`? A comment should document that.